test/cusparse/interfaces.jl

```diff
@@ -371,7 +371,7 @@ using LinearAlgebra, SparseArrays
              end
         end
 
-        @testset "$triangle(CuSparseMatrixBSR) \ CuVector -- $elty" for elty in [Float32,Float64,ComplexF32,ComplexF64]
+        @testset "$triangle(CuSparseMatrixBSR) \\ CuVector -- $elty" for elty in [Float32,Float64,ComplexF32,ComplexF64]
```

Comment:
I added four testsets with ` \`in the name. I forgot to use two `\\` when I updated them.